src/states_screens/race_result_gui.cpp

```diff
@@ -1325,11 +1325,14 @@ void RaceResultGUI::unload()
             : video::SColor(255, 255, 255, 255);
 
         unsigned int current_x = x;
+        if (RaceManager::get()->getNumPlayers() >= 10)
+        {
+            int pos_rank_width = m_font->getDimension(core::stringw(n + 1).c_str()).Width;
```

Comment:
Hi, I think _pos_rank_width_ should always be 2 digits to keep the names aligned.
```
1  john
2  john
10 john
11 john
```

(I only imagine it from the code. I only know a little bit c++. I programs c#/js...)

Reply:
Done